api/users/serializers.py

```diff
@@ -597,7 +606,11 @@ def get_resend_confirmation_url(self, obj):
             return '{}?resend_confirmation=true'.format(url)
 
     class Meta:
-        type_ = 'user_emails'
+        @staticmethod
+        def get_type(request):
+            if StrictVersion(request.version) < StrictVersion('2.15'):
```

Comment:
Big picture, `2.15` is hardcoded too many times, especially since 2.15 is no longer current.  This will at least need to be 2.16.   Consider defining this version in one place, `KEBAB_CASE_VERSION = 2.16` or similar (feel free to come up with a better name, and use that everywhere instead of the hardcoded 2.16.   Also, add a comment above where you define `KEBAB_CASE_VERSION` explaining its significance.